app/controllers/taxonomy_signups_controller.rb

```diff
@@ -45,6 +47,10 @@ def load_taxon
       .content_item(taxon_path)
   end
 
+  def validate_taxon_document_type
+    redirect_to '/' and return unless @taxon['document_type'] == 'taxon'
```

Comment:
Can taxon be `nil` or in any other way be an object that does not respond to `[]` or is it just a JOSN object?